packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -85,9 +84,8 @@ export default class SourceEditing extends Plugin {
 			const buttonView = new ButtonView( locale );
 
 			buttonView.set( {
-				label: t( 'Edit source' ),
-				// TODO: use icon
-				// icon: sourceEditingIcon,
+				label: t( 'Source' ),
+				icon: sourceEditingIcon,
 				tooltip: true,
 				withText: true,
 				class: 'ck-source-editing'
```

Comment:
`ck-source-editing-button` please. `ck-source-editing` is the name of the entire feature and this is super–vague.

Reply:
Same with the CSS name of `<textarea>` (`.source-editing`). It should be `.ck-source-editing-area`.